.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Bug report
+about: Report bugs in the software
+title: ''
+labels: bug, to be assigned
+assignees: ''
+
+---
+
+<!-- Provide a general summary of the issue in the title above.
+
+When adding program output or code snippets in the issue, use code blocks with
+appropriate syntax highlighting. -->
+
+#### Description
+<!-- Describe the issue briefly. -->
+
+##### Relates to
+<!-- Add links to any related issue or other references here. -->
+
+#### Expected behaviour
+<!-- Describe what is expected to happen. -->
+
+#### Current behaviour
+<!-- Describe what is actually happening. -->
+
+#### Reproducing the behaviour
+<!-- Describe a set of steps to reproduce the bug. -->
+
+1. 
+2. 
+3.  
+
+##### Software version
+<!-- Tell us about the version of software your using. You can also provide the
+git commit id, if you had built it from the source. -->
+
+##### Environment information
+<!-- Paste the output from `go env` command in the code block below. -->
+
+<details>
+
+<summary> Expand to see output from `go env` command.</summary>
+
+``` bash
+# Output of "go env" command
+```
+
+</details>
+
+#### Severity
+<!-- Tell us how severe the bug is. You can uncomment one of the following
+options: -->
+
+<!-- Trivial:&ensp;Has no real impact.<br> -->
+<!-- Minor:&ensp;Inconvinient for user, but does not break any feature.<br> -->
+<!-- Major:&ensp;Breaks one or more features or specifications.<br> -->
+<!-- Critical:&ensp;Causes the software to crash.<br> -->
```

Comment:
What do you think about just putting the description in parantheses, like `Trivial (has no real impact)`. Also, do we need the following `<br>`s?

Reply:
The descriptions are now put within paratheses and `<br>`s are removed, as in practice only one of the lines will be uncommented.
See this commit - 1c2dcf9403c97c826b7fa701b7398c9a3a379363.